.github/workflows/reusable_packaging_and_deployment.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           conan user
           conan profile new --detect default
           conan profile update settings.compiler.libcxx=libstdc++11 default
+          conan remote add -f dice-group https://conan.dice-research.org/artifactory/api/conan/tentris
```

Comment:
non-blocking comment:  What do we need from here that is not available at conan center? We should have everything we use for rdf4cpp on conan center. 

If we add this here we also must mention in the readme that you need to add the remote.

Reply:
Updated the README. We need `dice-hash` which isn't on conancenter